### PR TITLE
Allow users to configure passports to support claimable funds

### PR DESCRIPTION
**Pull request by dvargasfuertes**
https://app.clarity.so/cabin/view/3039c279-2ee2-4da2-a604-dc1c23d5010c

### CODE REVIEW

File: packages/app/pages/index.tsx
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:101]: Not a big fan of this name

File: packages/app/pages/index.tsx
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:261]: sorry about this noise, I have an autoformatter in my code editor that formats on save

File: packages/contracts/contracts/Passport.sol
Comment (dvargasfuertes) [inline packages/contracts/contracts/Passport.sol]: I'm going to wait for that Royalty PR to merge and will rebase this PR on it

File: packages/contracts/contracts/PassportFactory.sol
Comment (dvargasfuertes) [inline packages/contracts/contracts/PassportFactory.sol]: I wonder what's the max number of parameters a constructor/method could handle in Solidity lol

**Review by dvargasfuertes — COMMENTED** [review]
ccing @ssaitejar96 for review

File: packages/contracts/contracts/PassportFactory.sol
Comment (ssaitejar96) [reply packages/contracts/contracts/PassportFactory.sol]: We have more parameters coming in lol - royalties and private

File: packages/contracts/contracts/Passport.sol
Comment (ssaitejar96) [reply packages/contracts/contracts/Passport.sol]: Updated the royalties PR.

File: packages/contracts/contracts/Passport.sol
Comment (ssaitejar96) [inline packages/contracts/contracts/Passport.sol]: When you are resolving conflicts with royalties PR on this line, please make sure to put claimable before royaltyPcnt as I see that you've used p[5] in the UI side code.

File: packages/app/pages/index.tsx
Comment (ssaitejar96) [inline packages/app/pages/index.tsx]: You can pass royaltyPcnt as 0 for now when you rebase with royalties PR.  I will be pushing a PR for UI side royalty changes once this PR is merged.

**Review by ssaitejar96 — APPROVED** [review]
Overall looks great! Just left a couple of comments for when you rebase with royalties PR.

File: packages/contracts/contracts/Passport.sol
Comment (dvargasfuertes) [reply packages/contracts/contracts/Passport.sol]: I'd rather keep the order in the `get` and `constructor` consistent. Could easily update the UI to use `p[6]`

**Comment by dvargasfuertes** [comment]
Gonna retap you for review to look over royaltyInfo changes in `Passport.sol`

File: packages/contracts/contracts/Passport.sol
Comment (ssaitejar96) [inline packages/contracts/contracts/Passport.sol]: This should be receiver = royaltyRecipient, right?

**Review by ssaitejar96 — CHANGES_REQUESTED** [review]
One blocking comment.

File: packages/contracts/contracts/Passport.sol
Comment (dvargasfuertes) [reply packages/contracts/contracts/Passport.sol]: see, that's why I rerequested review

**Review by ssaitejar96 — APPROVED** [review]

**Pull request merged**
